src/components/question/answer.cjsx

```diff
@@ -117,7 +118,7 @@ Answer = React.createClass
       </div>
 
     if @props.show_all_feedback and answer.feedback_html
-      feedback = <Feedback key='question-mc-feedback'>{answer.feedback_html}</Feedback>
+      feedback = <Feedback key='question-mc-feedback' children={answer.feedback_html} />
 
```

Comment:
Ah, yea, I just updated that when I was debugging before i realized what was going on with the Feedback object.